### PR TITLE
Create message package for dummy_perception_publisher

### PR DESCRIPTION
**Pull request by nnmm**
* I couldn't get message + library code to compile in one package, so this is a separate package for the messages
* Uses `unique_identifier_msgs` instead of `uuid_msgs`
* Using `ament_cmake_auto` doesn't appear to work for message packages, no generated headers or source files show up in the install directory.

### CODE REVIEW

File: simulator/util/dummy_perception_publisher_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+find_package(rosidl_default_generators REQUIRED)
+find_package(autoware_perception_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(unique_identifier_msgs REQUIRED)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "msg/InitialState.msg"
+  "msg/Object.msg"
+  DEPENDENCIES std_msgs geometry_msgs autoware_perception_msgs unique_identifier_msgs
```
Comment (fred-apex-ai) [inline simulator/util/dummy_perception_publisher_msgs/CMakeLists.txt:22]: Add in next line

      ADD_LINTER_TESTS

**Review by fred-apex-ai — CHANGES_REQUESTED** [review]

File: simulator/util/dummy_perception_publisher_msgs/CMakeLists.txt (hunk 1 above)
Comment (nnmm) [reply simulator/util/dummy_perception_publisher_msgs/CMakeLists.txt:22]: This results in spurious errors. Omitted here and also removed in dependency autoware_perception_msgs.

File: simulator/util/dummy_perception_publisher_msgs/package.xml [hunk 2]
```diff
@@ -20,6 +19,7 @@
   <depend>std_msgs</depend>
   <depend>unique_identifier_msgs</depend>
 
+  <member_of_group>rosidl_interface_packages</member_of_group>
```
Comment (fred-apex-ai) [inline simulator/util/dummy_perception_publisher_msgs/package.xml:22]: for the record: needed to fix the XML linter

**Review by fred-apex-ai — APPROVED** [review]

**Comment by nnmm** [comment]
Closing since I found out how to keep the message definitions in the same package as the library, like it was before.

**Pull request closed without merging**